src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/IverksettingService.kt

```diff
@@ -68,7 +69,7 @@ class IverksettingService(val taskRepository: TaskRepository,
     }
 
     private fun førstePubliseringsflytTask(iverksett: Iverksett) = when {
-        erIverksettingUtenVedtaksperioder(iverksett) -> VedtakstatistikkTask.TYPE
+        erIverksettingUtenVedtaksperioder(iverksett) -> OpprettOppfølgingsOppgaveTask.TYPE
```

Comment:
Hvorfor ønsker man en oppfølgningsoppgave hvis den er avslått?

Reply:
Fordi at lokalkontoret skal være bevisste på at brukeren har søkt og fått avslått søknaden sin

Reply:
Er dette dokumentert i slack/favro?

Ser ikke ut som den oppretter noen oppgave hvis søknaden er avslått:
```
    fun skalOppretteVurderHendelseOppgave(iverksett: Iverksett): Boolean {
        return when (iverksett.behandling.behandlingType) {
            BehandlingType.FØRSTEGANGSBEHANDLING -> true
            BehandlingType.REVURDERING -> {
                when (iverksett.vedtak.vedtaksresultat) {
                    Vedtaksresultat.INNVILGET -> aktivitetEllerPeriodeEndret(iverksett)
                    Vedtaksresultat.OPPHØRT -> true
                    else -> false
                }
            }
            else -> false
        }
    }

```

Reply:
TEA-7426. Den sjekken sier at det skal opprettes oppgaver for alle førstegangsbehandlinger (innvilget, opphørt, eller avslått)

Reply:
Godt poeng!

Reply:
Husk att det kan være fint å få med årsaken til endringen i commit message, og PR. Er litt enklere å skjønne behovet for endringen då. Husk link til favro og